CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 4.2.0 (2020-09-25)
+
+- Add the ability to limit the countries available for selection in the phone field component. [#68](https://github.com/blackbaud/skyux-phone-field/pull/68)
```

Comment:
Changed "add" to "added" and you're good!

Reply:
Fixed